runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-    * `created`: the container has been created but the user-specified code has not yet been executed
-    * `running`: the container has been created and the user-specified code is running
-    * `stopped`: the container has been created and the user-specified code has been executed but is no longer running
+    * `created`: the container process has neither exited nor executed the user-specified code
```

Comment:
> Its not user-specified code, we went over this many times.

As [I pointed out][1] after #466 was closed, all of the changes proposed by #466 are already live.  So I'd rather keep the currently-live “user-specified code” language.  If you feel like that language is unclear, adjusting to something else seems orthogonal to the stopped → exists fix that this PR is focused on.  Maybe open a different PR?

[1]: https://github.com/opencontainers/runtime-spec/pull/466#issuecomment-258569376